packages/roosterjs-editor-core/lib/corePlugins/CopyPastePlugin.ts

```diff
@@ -162,16 +163,38 @@ export default class CopyPastePlugin implements PluginWithState<CopyPastePluginS
         return div;
     }
 
-    private cleanUpAndRestoreSelection(tempDiv: HTMLDivElement, range: Range, isCopy: boolean) {
-        if (isCopy && Browser.isAndroid) {
-            range.collapse();
+    private cleanUpAndRestoreSelection(
```

Comment:
Had to modify this because after copy/paste we were modifying from Table Selection to Normal Selection